core/src/test/scala/unit/kafka/log/LogCleanerIntegrationTest.scala

```diff
@@ -138,6 +139,8 @@ class LogCleanerIntegrationTest(compressionCodec: String) {
       logProps.put(LogConfig.SegmentIndexBytesProp, 100*1024: java.lang.Integer)
       logProps.put(LogConfig.FileDeleteDelayMsProp, deleteDelay: java.lang.Integer)
       logProps.put(LogConfig.CleanupPolicyProp, LogConfig.Compact)
+      logProps.put(LogConfig.MinCleanableDirtyRatioProp, 0F: java.lang.Float)
```

Comment:
Actually can we make this configurable in ```makeCleaner``` (default to 0.0f) and replace ```minDirtyMessages``` which is not used any more? I think in an older version we use minDirtyMessages but changed it to a ratio, but never changed this function..

Also we can update the comment of "wait until we clean up to base_offset of active segment - minDirtyMessages" above.